pkg/frontend/querymiddleware/codec_test.go

```diff
@@ -205,7 +206,7 @@ func TestLabelsQueryRequest(t *testing.T) {
 			expectedGetEndOrDefault:   1708588800 * 1e3,
 		},
 		{
-			name: "label names with start timestamp, no end timestamp, multiple matcher sets",
+			name: "label names with start and end timestamp, multiple matcher sets",
```

Comment:
noticed that the test description here and below doesn't match the actual test case. Confirmed with @francoposa that it's the description that's wrong not the implementation